### ISSUE TITLE
Abstract support?

### ISSUE THREAD

**Issue by kevinresol** [haxetink/tink_syntaxhub#7]
[Currently it only match classes / interfaces](https://github.com/haxetink/tink_syntaxhub/blob/b8a7e51/src/tink/SyntaxHub.hx#L38)

But then ClassBuilder doesn't work with `TAbstract`...

Currently it does match the impl class, but there I think the class meta info is lost.

**Comment by back2dos** [comment]
Yeah, in principle I wouldn't mind, but there's that ClassBuilder dependency that's kind of hard to deal with. Ideas?

**Comment by kevinresol** [comment]
Can we just get rid of `ClassBuilder` and pass the `Type` directly?
And add `Types.getClassBuilder(type:Type)` for quick creation.

**Comment by back2dos** [comment]
Probably, yes. I think there was some obscure reason why I went with ClassBuilder in the first place, but I can't remember right now.
